pkg/container-utils/containerutils.go

```diff
@@ -181,3 +181,41 @@ func GetIfacePeers(pid int) ([]*net.Interface, error) {
 
 	return ifacesHost, err
 }
+
+// GetIfaces returns the interfaces inside the container
+// It returns only veth and up devices
+func GetIfaces(pid int) ([]*net.Interface, error) {
```

Comment:
Is it only for tests?

If so, it could be moved to testutils.

If not, what about moving this in `pkg/utils/netns/netns.go` to solve the import cycle?

Also, this is only used to get the IP so it could return a `[]net.IP`.